README.md

```diff
@@ -9,7 +9,7 @@ This repository contains the source for the [Rundeck](http://rundeck.org/) [dock
 1. Supervisor, Apache2, and rundeck
 1. No SSH.  Use docker exec or [nsenter](https://github.com/jpetazzo/nsenter)
 1. If RUNDECK_PASSWORD is not supplied, it will be randomly generated and shown via stdout.
-1. Supply the SERVER_URL or else you won't get too far :)
+1. Supply the SERVER_URL or it will default to https://0.0.0.0:44443
```

Comment:
https://github.com/jjethwa/rundeck/blob/master/content/opt/run#L30

But you actually default to https://0.0.0.0:4443 in the /opt/run script...